tesla_smart_charger/routes/auth_routes.py

```
@@ -325,10 +343,13 @@ def _perform_token_exchange(code: str, state: str, issuer: Optional[str] = None)
         )
         api = TeslaAPI(tmp_vehicle)
         vehicles_list = api.get_vehicles()
-    except Exception as exc:
-        tsc_logger.warning("Could not list vehicles after OAuth: %s", exc)
+    # Deliberately broad: failing to list vehicles must not fail the OAuth
+    # exchange itself — the tokens are still valid and already stored below.
+    # Logged at WARNING (not ERROR) since this path is non-fatal.
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)